cli/network.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/alibaba/pouch/apis/types"
+
+	"github.com/spf13/cobra"
+)
+
+// NetworkCommand is used to implement 'network' command.
+type NetworkCommand struct {
+	baseCommand
+}
+
+// Init initializes NetworkCommand command.
+func (n *NetworkCommand) Init(c *Cli) {
+	n.cli = c
+
+	n.cmd = &cobra.Command{
+		Use:   "network [command]",
+		Short: "Manage pouch networks",
+		Args:  cobra.MinimumNArgs(1),
+	}
+
+	c.AddCommand(n, &NetworkCreateCommand{})
+}
+
+// Run is the entry of NetworkCommand command.
+func (n *NetworkCommand) Run(args []string) {}
+
+// NetworkCreateCommand is used to implement 'network create' command.
+type NetworkCreateCommand struct {
+	baseCommand
+
+	name       string
+	driver     string
+	gateway    string
+	ipRange    string
+	ipamDriver string
+	ipamOpts   []string
+	subnet     []string
+	options    []string
+	labels     []string
+}
+
+// Init initializes NetworkCreateCommand command.
+func (n *NetworkCreateCommand) Init(c *Cli) {
+	n.cli = c
+
+	n.cmd = &cobra.Command{
+		Use:   "create <NAME>",
+		Short: "Create a pouch network",
+	}
+
+	flagSet := n.cmd.Flags()
+
+	flagSet.StringVarP(
+		&n.name,
+		"name",
+		"n",
+		"",
+		"the name of network")
+	flagSet.StringVarP(
+		&n.driver,
+		"driver",
+		"d",
+		"bridge",
+		"the driver of network")
+	flagSet.StringVar(
+		&n.gateway,
+		"gateway",
+		"",
+		"the gateway of network")
+	flagSet.StringVar(
+		&n.ipRange,
+		"ip-range",
+		"",
+		"the range of network's ip")
+	flagSet.StringSliceVar(
+		&n.subnet,
+		"subnet",
+		nil,
+		"the subnet of network")
+	flagSet.StringVar(
+		&n.ipamDriver,
+		"ipam-driver",
+		"",
+		"the ipam driver of network")
+	flagSet.StringSliceVarP(
+		&n.options,
+		"option",
+		"o",
+		nil,
+		"create network with options")
+	flagSet.StringSliceVarP(
+		&n.labels,
+		"label",
+		"l",
+		nil,
+		"create network with labels")
+}
+
+// Run is the entry of NetworkCreateCommand command.
+func (n *NetworkCreateCommand) Run(args []string) {
+	name := n.name
+	if len(args) != 0 {
+		name = args[0]
+	}
+	if name == "" {
+		fmt.Fprintf(os.Stderr, "Need to specified network's name\nSuch as: pouch network create <NAME> [flags]\n")
+		return
+	}
+
+	networkRequest := &types.NetworkCreateRequest{
+		Name: name,
+	}
+
+	networkCreate := &types.NetworkCreate{
+		Driver: n.driver,
+	}
+
+	networkRequest.NetworkCreate = networkCreate
+
+	apiClient := n.cli.Client()
+	resp, err := apiClient.NetworkCreate(networkRequest)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to create network: %s, err: %v\n", n.name, err)
```

Comment:
We do not use this way any more. Now we just return err, in https://github.com/alibaba/pouch/blob/master/main.go#L39 we will output error in STDERR.